stemcell_builder/stages/bosh_go_agent/assets/restrict-nats-api-access-allow-monit-api

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# THESE SCRIPTS ARE EXECUTED IN LEXICAL ORDER. MAKE SURE TO CHECK IF ANYTHING ELSE (CURRENTLY MONIT) WRITES OTHER CGROUP BASED RULES TO AVOID INTERFERENCE. IT'S PROBABLY AGOOD IDEA TO -A DROPS AND -I ACCEPTS
```

Comment:
changes in https://github.com/cloudfoundry/bosh-linux-stemcell-builder/pull/197

Reply:
Are those changes going to get merged into the `bionic` branch? (I ask because I still am unclear about what we're supposed to be doing with the `bionic` branch.)

Reply:
yes this will be merged in bionic/master
and later when we do a release for 1.x we will cherry pick/rebase bionic/master in to bionic/1.x